lib/PHPCfg/Parser.php

```diff
@@ -216,6 +216,11 @@ protected function parseTypeNode(?Node $node): Op\Type
         throw new \LogicException("Unknown type node: " . $node->getType());
     }
 
+    protected function parseStmt_Block(Stmt\Block $node)
+    {
+        $this->block->children[] = new Op\Stmt\Block($this->parseNodes($node->stmts, new Block($this->block)));
```

Comment:
We should flatten this into the parent block, instead of creating a new statement for it.